### ISSUE TITLE
Revise Concept-Graph generation

### ISSUE THREAD

**Issue by sjcjoosten** [AmpersandTarski/Ampersand#120]
Current graphviz concept graph is not so pretty. It should be improved.

**Comment by hanjoosten** [comment]
This is a type of picture that was generated by RAP2

![pat_95inhuurprocedure](https://cloud.githubusercontent.com/assets/1159672/15960427/09537008-2f00-11e6-8049-3c9ddc70468f.png)

**Comment by sjcjoosten** [comment]
Not bad, does readability improve when using the left-to-right or top-to-bottom layout of dot instead of neato?

**Comment by hanjoosten** [comment]
There is a tool called [CMAP](https://cmap.ihmc.us/) that seems to have nice pictures. However, it is not open source, and there seems to be no API. 
An inspiring picture I found on their site:

![cmapimplementinacourse](https://user-images.githubusercontent.com/1159672/30480827-0cdc5d56-9a1c-11e7-8aae-fded9baeb772.jpg)

**Comment by hanjoosten** [comment]
I am trying to get better images. In search of this, I stubled upon a bug in graphviz. I made [an issue](https://gitlab.com/graphviz/graphviz/issues/1485).

**Comment by Michiel-s** [comment]
Another visualization I use (not Haskell btw):

![image](https://user-images.githubusercontent.com/7961582/51741995-bb3d8600-2098-11e9-9301-2c06994cb0b3.png)

VOWL: http://www.visualdataweb.de/webvowl/

**Comment by hanjoosten** [comment]
With the new approach I made the following changes:

- The inbetween node for all relations has been removed
- endo relations are drawn by using an invisible node. A line is drawn between that invisible node and the source/target concept of that relation. This gives better results than drawing the relation as a loop. 
- A border is drawn for concept nodes. 
- The graph is renderd using `neato`. Using `dot` I wasn't able to control the length of edges at all. Neato gives a limited control of the length, wich is better than nothing at all. 

This results in pictures like this one:

![relationsinpatternarbeidsduur](https://user-images.githubusercontent.com/1159672/51743537-3a34bd80-209d-11e9-9039-114d3aadf50d.png)
